README.md

```diff
@@ -88,3 +88,163 @@ In order to configure connection to elasticsearch you have to provide following
+| `electronicAccess`                        | full text | `electronicAccess any "resource"`                 | An alias for all `electronicAccess` fields - `uri`, `linkText`, `materialsSpecification`, `publicNote`|
+| `electronicAccess.uri`                    | term      | `electronicAccess.uri="http://folio.org*"`        | Search by electronic access URI|
+| `electronicAccess.linkText`               | full text | `electronicAccess.linkText="Folio website"`       | Search by electronic access link text |
+| `electronicAccess.materialsSpecification` | full text | `electronicAccess.materialsSpecification="book"`  | Search by electronic access material specification |
+| `electronicAccess.publicNote`             | full text | `electronicAccess.publicNote="a rare book"`       | Search by electronic access public note |
+| `staffSuppress`                           | term      | `staffSuppress==true`                             | Matches instances that are staff suppressed |
+| `discoverySuppress`                       | term      | `discoverySuppress==true`                         | Matches instances that are suppressed from discovery|
+| `publicNotes`                             | full text | `publicNotes all "public note"`                   | Matches instances that have a public note (i.e. `note.staffOnly` is `false`) |
+| `isbn`                                    | term      | `isbn="1234*943"`                                 | Matches instances that have an ISBN  identifier with the given value |
+| `issn`                                    | term      | `issn="1234*943"`                                 | Matches instances that have an ISSN  identifier with the given value |
+
+
+#### Holdings-records search options
+
+| Option                        | Type |Example                                    | Description                  |
+| :-----------------------------|:----:| :-----------------------------------------|:------------------------------|
+| `holdings.id`                 | term | `holdings.id=="1234567"`                  | Matches instances that have a holding with the id |
+| `holdings.permanentLocationId`| term | `holdings.permanentLocationId=="123765"`  | Matches instances that have holdings with given permanentLocationId |
+| `holdings.discoverySuppress`  | term | `holdings.discoverySuppress==true`        | Matches instances that have holdings suppressed/not suppressed from discovery |
+| `holdings.hrid`               | term | `holdings.hrid=="hr10*3"`                 | Matches instances that have a holding with given HRID |
+| `holdingTags`                 | term | `holdingTags=="important"`                | Matches instances that have holdings with given tags |
+| `holdingsFullCallNumbers`     | term | `holdingsFullCallNumbers="cn*434"`        | Matches instances that have holdings with given call number string (prefix + call number + suffix) |
```

Comment:
electronic access for holdings and items are missing

Reply:
Thanks, added.